config/zones.json

```
@@ -655,6 +655,104 @@
             }
         ],
         "ambient_npcs": [
+            {
+                "comment": "Green Starfighter",
+                "model_id": 721,
+                "npc_type": 2,
+                "pos_x": 118.403900,
+                "pos_y": 56.782230,
+                "pos_z": -328.281600,
+                "pos_w": 1.0,
+                "rot_x": 0.0,
+                "rot_y": 0.0,
+                "rot_z": 0.0,
+                "is_spawned": false,
+                "enable_gravity": false,
+                "tickable_procedures": {
+                    "spawn": {
+                        "weight": 36,
+                        "steps": [
+                            {
+                                "spawned_state": true,
+                                "rail_id": 36,
+                                "duration_millis": 0
+                            }
+                        ],
+                        "next_possible_procedures": ["remove"]
+                    },
+                    "remove": {
+                        "steps": [
+                            {
+                                "composite_effect_id": 1021,
+                                "duration_millis": 12000
+                            },
+                            {
+                                "spawned_state": false,
+                                "enable_graceful_removal": false,
+                                "duration_millis": 1000
+                            }
+                        ],
+                        "next_possible_procedures": ["spawn"]
```

```suggestion
                        "next_possible_procedures": ["spawn", "empty"]
```
I don't think the `empty` procedure can ever be executed after the first spawn with the current config. I also think `empty`'s next possible procedure should be `spawn`